index.html

```diff
@@ -339,128 +325,119 @@ <h1>Important Terms</h1>
 			</dd>
-			<dt><dfn data-export="" data-lt="relationship|relationships">Relationship</dfn></dt>
+			<dt><dfn data-lt="relationship|relationships">Relationship</dfn></dt>
 			<dd>
 			  <p>A connection between two distinct things. Relationships may be of various types to indicate which <a>object</a> labels another, controls another, etc.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="role|roles">Role</dfn></dt>
+			<dt><dfn data-lt="role|roles">Role</dfn></dt>
 			<dd>
 			  <p>Main indicator of type. <!-- (removing, vague) The object's role is the class of <a class="termref" data-lt="object">objects</a> of which it is a member. --> This <a class="termref" data-lt="semantics">semantic</a> association allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.</p>
 			</dd>
-			<dt><dfn data-export="">Root <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> node</dfn></dt><!--Not used-->
+			<dt><dfn>Root <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> node</dfn></dt><!--Not used-->
 			<dd>
 			  <p> The primary element containing non-metadata content. In many languages, this is  the document element but in <abbr title="Hypertext Markup Language">HTML</abbr>, it is the <code>&lt;body&gt;</code>.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="semantics|semantic|semantically">Semantics</dfn></dt>
+			<dt><dfn data-lt="semantics|semantic|semantically">Semantics</dfn></dt>
 			<dd>
-			  <p>The meaning of something as understood by a human, defined in a way that computers can process a representation of an <a>object</a>, such as <a class="termref" data-lt="element">elements</a> and <a class="termref" data-lt="attribute">attributes</a>, and reliably represent the object in a way that various humans will achieve a mutually consistent understanding of the object.</p>
+			  <p>The meaning of something as understood by a human, defined in a way that computers can process a representation of an <a>object</a>, such as <a class="termref" data-lt="element">elements</a> and [=attribute|attributes=], and reliably represent the object in a way that various humans will achieve a mutually consistent understanding of the object.</p>
 			</dd>
-			<dt><dfn data-export="" data-lt="state|states">State</dfn></dt>
+			<dt><dfn data-lt="state|states">State</dfn></dt>
```

Comment:
Needs export

Reply:
Exporting. I think HTML might have something similar. 
https://html.spec.whatwg.org/#states-of-the-type-attribute

We should try to align those.